addon/components/o-s-s/information-section.stories.js

```diff
@@ -0,0 +1,116 @@
+import { hbs } from 'ember-cli-htmlbars';
+
+export default {
+  title: 'Components/OSS::InformationSection',
+  argTypes: {
+    title: {
```

Comment:
you can be a little more precise with the argTypes in storybook, there are examples in other files in the project
```javascript
      table: {
        type: { summary: 'string' },
        defaultValue: { summary: 'undefined' }
      },
      control: {
        type: 'text'
      }
```

Reply:
Specified argTypes in sb

Reply:
I think the `defaultValue`s are wrong ^^
should be 'undefined' everywhere no ? 🤷